### PR TITLE
Update domains-faq.yml

### PR DESCRIPTION
**Pull request by msbemba**
Updated FAW with a question about personal and family offerings

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7662

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/setup/domains-faq.yml [hunk 1]
```diff
@@ -69,6 +69,12 @@ sections:
           Using a custom domain like "**rob\@contoso.com**" with Microsoft 365 can help build credibility and recognition for your brand. 
             
           You can [buy a domain in Microsoft 365 and we'll set it up automatically](../get-help-with-domains/buy-a-domain-name.md), or you can buy or bring one you already own from a domain registrar.
+          
+      - question: Is it possible to connect a domain with Personal or Family offerings?
+        answer: |
+           As a benefit of your Microsoft 365 Family or Microsoft 365 Personal subscription, you can create a personalized email address that's associated with your Outlook.com mailbox. For example, yourname@example.com. At the moment, we only support connecting domains managed by GoDaddy with Outlook.com.
+
+         - [Personalized emailaddress in Microsoft 365 Family or Microsoft 365 Personal subscription](https://support.microsoft.com/en-us/office/get-a-personalized-email-address-in-microsoft-365-75416a58-b225-4c02-8c07-8979403b427b)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/domains-faq.yml]:
```suggestion
           As a benefit of your Microsoft 365 Family or Microsoft 365 Personal subscription, you can create a personalized email address that's associated with your Outlook.com mailbox, for example, yourname@example.com. At the moment, we only support connecting domains managed by GoDaddy with Outlook.com. For more information, see [Get a personalized email address in Microsoft 365](https://support.microsoft.com/office/get-a-personalized-email-address-in-microsoft-365-75416a58-b225-4c02-8c07-8979403b427b).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks

**Pull request merged**